core/amber/build.sbt

```diff
@@ -62,7 +62,29 @@ val dropwizardDependencies = Seq(
   "org.glassfish.jersey.media" % "jersey-media-multipart" % jerseyMultipartVersion,
   "com.fasterxml.jackson.module" % "jackson-module-jsonSchema" % jacksonVersion,
   "com.fasterxml.jackson.module" % "jackson-module-scala_2.12" % jacksonVersion,
-  "com.kjetland" % "mbknor-jackson-jsonschema_2.12" % "1.0.39"
+)
+
+
+
+// deps from library
+//"com.kjetland" % "mbknor-jackson-jsonschema_2.12" % "1.0.39"
+
+val jacksonModuleScalaVersion = "2.12.1"
```

Comment:
are we having duplicate dependencies (we have jackson above)?

Reply:
fixed